### ISSUE TITLE
模型预测问题

### ISSUE THREAD

**Issue by kleinba1** [foranonymous123/CNNforFaultInterpretation#11]
您好，我先是使用了generatePatchipyTrainValTest.ipynb里的代码训练出model文件然后使用savePredNpy_thebetest.ipynb进行预测，但效果图基本上是混沌的状态，后来尝试选用您提供的unet_96_48_seed_test.model模型预测，依旧是很混沌没有形成预测断层的状态，可以请教一下我大概是哪个部分出现问题需要调整吗

**Comment by Hyul48** [comment]
Have you solved that problem? I encountered the issue too

**Comment by anyuzoey** [comment]
Could you explain your question in a bit more context? I'm confused and dont know how to help.

**Comment by kleinba1** [comment]
> Have you solved that problem? I encountered the issue too

Yes, I have resolved the issue. The problem I encountered was likely caused by a mismatch between the CUDA version and the driver. After reinstalling CUDA and configuring the environment, the expected result appeared. I hope this can be of some help to you.

**Comment by kleinba1** [comment]
> Could you explain your question in a bit more context? I'm confused and dont know how to help.

感谢您的回复，抱歉没能及时看到QAQ，这个问题我在重新配置环境后已经解决。我需要对原始数据进行处理，可以问一下我可以通过什么途径获取segy数据吗？

**Comment by anyuzoey** [comment]
> > Could you explain your question in a bit more context? I'm confused and dont know how to help.
> 
> 感谢您的回复，抱歉没能及时看到QAQ，这个问题我在重新配置环境后已经解决。我需要对原始数据进行处理，可以问一下我可以通过什么途径获取segy数据吗？

这个你看看能不能下载：https://drive.google.com/drive/folders/13cUUbr9eHm4DVD_bDJXv_NSQFk-lO_hJ?usp=sharing

**Comment by kleinba1** [comment]
> > > Could you explain your question in a bit more context? I'm confused and dont know how to help.
> > 
> > 
> > 感谢您的回复，抱歉没能及时看到QAQ，这个问题我在重新配置环境后已经解决。我需要对原始数据进行处理，可以问一下我可以通过什么途径获取segy数据吗？
> 
> 这个你看看能不能下载：https://drive.google.com/drive/folders/13cUUbr9eHm4DVD_bDJXv_NSQFk-lO_hJ?usp=sharing

正在下载FILE_000001.sgy，可以下载，非常感谢！

**Comment by kleinba1** [comment]
> > > Could you explain your question in a bit more context? I'm confused and dont know how to help.
> > 
> > 
> > 感谢您的回复，抱歉没能及时看到QAQ，这个问题我在重新配置环境后已经解决。我需要对原始数据进行处理，可以问一下我可以通过什么途径获取segy数据吗？
> 
> 这个你看看能不能下载：https://drive.google.com/drive/folders/13cUUbr9eHm4DVD_bDJXv_NSQFk-lO_hJ?usp=sharing

您好，我下载后压缩包解压提示文件已损坏，修改后缀后读取报错Unable to determine the endianness of the file.

**Comment by anyuzoey** [comment]
会不会是gdrive下载压缩包中间中断了啊，这个链接之前分享给别人没说过有问题。麻烦再试一下呢？这个数据在学校的一个硬盘里，重新上传有点费劲～ 祈祷🙏

**Comment by kleinba1** [comment]
> 会不会是gdrive下载压缩包中间中断了啊，这个链接之前分享给别人没说过有问题。麻烦再试一下呢？这个数据在学校的一个硬盘里，重新上传有点费劲～ 祈祷🙏

这两天在离校回家的路上，抱歉没能及时回复，后面尝试了几次都出现了下载中断的问题，目前还没顺利完整下载排除gdrive下载问题，我继续尝试尝试，提前祝您春节愉快啦
